plugins/wasm-cpp/extensions/basic_auth/README_EN.md

```diff
@@ -0,0 +1,117 @@
+<p>
+   English | <a href="README.md">中文</a>
+</p>
+
+# Description 
+`basic-auth` plugin implements the function of authorization based on the HTTP Basic Auth standard.
+
+# Configuration Fields
+
+| Name        | Type            | Requirement | Default Value | Description                                                 |
+| ----------- | --------------- | -------- | ------ | ---------------------------------------------------- |
+| `consumers` | array of object | Required     | -      | Caller of the service for authentication of requests. |
+| `_rules_`   | array of object | Optional     | -      | Configure access permission list for specific routes or domains to authenticate requests.: |
+
+Filed descriptions of `consumers` items:
+
+| Name         | Type   | Requirement | Default Value | Description                           |
+| ------------ | ------ | ----------- | ------------- | ------------------------------------- |
+| `credential` | string | Required    | -             | Credential for this consumer's access |
+| `name`       | string | Required    | -             | Name of this consumer                 |
+
+Configuration field descriptions for each item in `_rules_` are as follows:
+
+| Field Name            | Data Type        | Requirement                                          | Default | Description                                               |
+| ---------------- | --------------- | ------------------------------------------------- | ------ | -------------------------------------------------- |
+| `_match_route_`  | array of string | One of `_match_route_` or `_match_domain_` | -      | Configures the routes to match for request authorization.                               |
+| `_match_domain_` | array of string | One of `_match_route_` , `_match_domain_` | -      | Configures the domains to match for request authorization.                                   |
```

Comment:
Here comes "configures". But in line 13, it's "configure". Which one should we use here?